### DISCUSSION TITLE
winget-intune cli v0.5.x - feedback please

### DISCUSSION THREAD

**Discussion by svrooij** [svrooij/WinTuner#12] [Announcements]
Two weeks ago I [released](https://github.com/svrooij/WingetIntune/discussions/5) the first version of this tool and your responses on LinkedIn and Twitter are overwhelming.

Today I'm happy to announce version `0.5.5`, follow the [install/update](https://github.com/svrooij/WingetIntune#installing) instructions.

## Features 0.5.1 📃

- Support for categories (you can now set the `--category "{categoryName}"` parameter and the app is added to that category in the Company Portal.
- Support for assigning apps upon publishing. Add the `--available`, `--required` or `--uninstall` parameter followed by `allusers`, `alldevices` or a group id, to the publish command and the app is automatically added to that group.
- AzCopy has been replaced by a [ChunkedAzureFileUploader](https://github.com/svrooij/WingetIntune/blob/main/src/WingetIntune/Implementations/ChunkedAzureFileUploader.cs) and now all uploads succeed.
- Only a single login is needed for uploading. In cases where you were using an account not connected to windows you would get multiple login popups.

### Additional improvements ✔️:

- The app is less verbose (by default), you can see all the requests by using the `--verbose` parameter
- There is support for json logging, in case you want to parse the output `--json`
- I added nerd fonts to the gifs (as instructed by [Jan](https://twitter.com/jandedobbeleer/status/1692206404893880470?s=20))
- Ignoring undefined yaml properties from manifest, instead of crashing

### Version 0.5.2:

- Support for French windows versions. This app depends upon `winget` and they translate the output to the users' locale... 😒

### Version 0.5.3-0.5.4

- Removed spaces from Installer filename [pr 18](https://github.com/svrooij/WingetIntune/pull/18)
- Added the now mandatory `setupFilePath` for Win32LobApps. [pr 19](https://github.com/svrooij/WingetIntune/pull/19)

### Version 0.5.5

- 🪲Incorrect argument order for login [pr 20](https://github.com/svrooij/WingetIntune/pull/20/files)

![publish-app-in-seconds-intune](https://github.com/svrooij/WingetIntune/assets/1292510/ff06d0f5-caa9-40db-b67f-8e5832fda8ec)

## Feedback needed 💪⭐

I really want to know what you think about this tool, so please:

- Leave a message below 👇
- Send me a tweet `@svrooij`
- Add a comment on linkedin under posts regarding this tool

**Comment by Wizkeezy** [comment]
Hi Svrooij,

im getting this error, can you help to resolve it?

>winget-intune package Zoom.Zoom --package-folder c:\temp
info: WingetIntune.Implementations.WingetManager[4] Getting package info for Zoom.Zoom (null)
info: WingetIntune.Commands.PackageCommand[0] Package (null) (null) from Winget
**Unhandled exception: System.ArgumentNullException: Value cannot be null. (Parameter 'id')….

**Reply by svrooij** [reply]
Please check https://github.com/svrooij/WingetIntune/issues/new/choose for creating a valid issue, there is a form and everything
